go/teams/chain.go

```diff
@@ -2196,10 +2208,21 @@ func (t *teamSigchainPlayer) updateInvites(stateToUpdate *TeamSigChainState, add
 	}
 }
 
-func (t *teamSigchainPlayer) completeInvites(stateToUpdate *TeamSigChainState, completed map[keybase1.TeamInviteID]keybase1.UserVersionPercentForm) {
+func (t *teamSigchainPlayer) completeInvites(stateToUpdate *TeamSigChainState, completed map[keybase1.TeamInviteID]keybase1.UserVersionPercentForm) error {
 	for id := range completed {
+		invite, ok := stateToUpdate.inner.ActiveInvites[id]
+		if !ok {
+			// TODO: That's likely an error, but we've allowed this for so long
+			// that we have to prove that no team has done this in order to
+			// change this.
+			continue
+		}
+		if invite.MaxUses != nil {
```

Comment:
Do we wanna allow completed_invites for invites with etime but not MaxUses? it is ok for me if so, but I thought new invites would only use UsedInvites.

Reply:
I think it's fine to tie used_invites only for max_uses. Now that I'm reading it, it's not 100% clear from the doc, but I sort of arrived at this when working on this PR. 

It makes sense to only force you into "used_invites" multi-map when an invite is multiple use, for everything else completed_invites does the job.